src/components/calcite-value-list-item/calcite-value-list-item.tsx

```diff
@@ -81,22 +96,51 @@ export class CalciteValueListItem {
 
   getPickListRef = (el) => (this.pickListItem = el as HTMLCalcitePickListItemElement);
 
+  handleKeyDown = (event: KeyboardEvent) => {
+    if (event.key === " ") {
+      this.handleActivated = !this.handleActivated;
+    }
+  };
+
+  handleBlur = () => {
+    this.handleActivated = false;
+  };
+
   // --------------------------------------------------------------------------
   //
   //  Render Methods
   //
   // --------------------------------------------------------------------------
 
+  renderHandle() {
+    const { icon } = this;
+    if (icon === ICON_TYPES.grip) {
+      return (
+        <a
+          class={{ [CSS.handle]: true, [CSS.handleActivated]: this.handleActivated }}
+          tabindex="0"
+          data-js-handle="true"
```

Comment:
question: What is this data attribute for?

Reply:
It's used as a hook to verify the keyDown event originated from this element.
https://github.com/Esri/calcite-app-components/blob/hrobbins/valueList-dragDrop-keyboard-support-356/src/components/calcite-value-list/calcite-value-list.tsx#L207-L213